doc/manuals/orion-api.md

```diff
@@ -2991,7 +3015,7 @@ A context registration is represented by a JSON object with the following fields
 | `description`           | ✓        | string | Description given to this registration.                                                                                                                                                     |
 | [`provider`](#registrationprovider)              |          | object | Object that describes the context source registered.                                                                                                                                        |
 | [`dataProvided`](#registrationdataprovided)          |          | object | Object that describes the data provided by this source.                                                                                                                                     |
-| `status`       | ✓        | string | Enumerated field which captures the current status of this registration with the possibles values: [`active`, `inactive`, `expired` or `failed`]. If this field is not provided at registration creation time, new registrations are created with the `active` status, which may be changed by clients afterwards. For expired registrations, this attribute is set to `expired` (no matter if the client updates it to `active`/`inactive`). Also, for registrations experiencing problems with forwarding operations, the status is set to `failed`. As soon as the forwarding operations start working again, the status is changed back to `active`. |
```

Comment:
Pay attention to this

Reply:
Let's double check that active, inactive and expired are really working. Not fully sure...

Reply:
I have realized we say in the differences section:

> The `inactive` value for `status` is not supported

But nothing is told about the `failed` status.

Definitively, we should check how Orion works in this sense to precisely describe the status it supports (and how) for registrations in this document.